server/models/image_item.py

```diff
@@ -90,6 +91,11 @@ def _createLargeImageJob(self, item, fileObj, user, token):
             user=user)
         jobToken = Job.createJobToken(job)
 
+        outputName = os.path.splitext(fileObj['name'])[0] + '.tiff'
+        if outputName == fileObj['name']:
+            outputName = (os.path.splitext(fileObj['name'])[0] + '.' +
```

Comment:
Yes, Girder already has the time that the file was added to the system stored in the file metadata.